react-components/src/hooks/useAll3dModels.ts

```diff
@@ -0,0 +1,19 @@
+/*!
```

Comment:
Need to add Test file!

Reply:
https://github.com/cognitedata/reveal/pull/4746/commits/add9cc029ccbbc06fb84957f9036471ac74a4995
and 
https://github.com/cognitedata/reveal/pull/4746/commits/e84ecdaf5dd912b82ab39d2a9a48763381a66339

I am keeping the endpoints wrapped in functions to facilitate the mocking in utest instead of mocking the SDK. And also to make the functions usable to others if needed.